include/eve/module/algo/algo/copy_if.hpp

```diff
@@ -135,6 +135,9 @@ template<typename TraitsSupport> struct copy_if_ : TraitsSupport
 //!   ** Return value **
 //!
 //!    relaxed_iterator past the last written element.
+//!
+//!   @see `transform_copy_if`
+//!
```

Comment:
I don't think this is a good place.

After defined in the header

Note: if you need to apply a transformation, you can use eve::views::map view or eve:: algo::transform_copy_uf